### ISSUE TITLE
Remove unused, legacy function commit_poll

### ISSUE THREAD

**Issue by willingc** [openhatch/oh-mainline#1646]
In the `mysite/missions/svn/views.py` file, `commit_poll` is an unused function from the early years of OpenHatch missions. The function is no longer used by any of the existing code. I recommend removing the function and the `urls.py` redirection to it.

**Comment by paulproteus** [comment]
Are ​you sure that Javascript doesn't poll this periodically? I thought
that was how the svn mission advanced itself, when you get to the last step.

**Comment by willingc** [comment]
@paulproteus Well, not 100% sure. I'm sure that when I tested it locally that it did not get called. That said, when I tested the other fixes to the SVN mission that just got merged, they worked locally on both FF and Chrome with no problems. Yet after deployed, the form box doesn't display until after you hit submit for the diff page on FF, and on Chrome, it's obscured by the footer :(

One step forward...two steps back :question:
